compat.go

```diff
@@ -273,24 +268,32 @@ func compatMetricConfig(c *global.MetricConfig) *config.MetricConfig {
 		return nil
 	}
 	return &config.MetricConfig{
-		Enable:      c.Enable,
-		Port:        c.Port,
-		Path:        c.Path,
-		Prometheus:  compatMetricPrometheusConfig(c.Prometheus),
-		Aggregation: compatMetricAggregationConfig(c.Aggregation),
-		Protocol:    c.Protocol,
+		Enable:             c.Enable,
+		Port:               c.Port,
+		Path:               c.Path,
+		Prometheus:         compatMetricPrometheusConfig(c.Prometheus),
+		Aggregation:        compatMetricAggregationConfig(c.Aggregation),
+		Protocol:           c.Protocol,
+		EnableMetadata:     c.EnableMetadata,
+		EnableRegistry:     c.EnableRegistry,
+		EnableConfigCenter: c.EnableConfigCenter,
+		EnableRpc:          c.EnableRpc,
```

Comment:
Shouldn’t EnableRpc be deleted?

Reply:
Yeah, but I worry it will introduce new issues. So we can do that in a separate patch later.

Reply:
OK